prime-router/src/main/kotlin/cli/tests/Simulator.kt

```diff
@@ -369,6 +406,69 @@ class Simulator : CoolTest() {
 //            results += productionSimulation(environment, options)
 //            results += productionSimulation(environment, options)
 //        }
-        return teardown(results, elapsedTime)
+        return teardown(results, elapsedTime, startTime, options.asyncProcessMode)
+    }
+
+    /**
+     * Checks that at least the correct number of process records are in the Action table after running simulator.
+     * There may be more than expected if other tests occur at the same time, but there should not be fewer.
+     */
+    private suspend fun checkTimedResults(
+        expectedResults: Int,
+        afterDateTime: OffsetDateTime,
+        taskToCheck: TaskAction,
+        maxPollSecs: Int = 180,
+        pollSleepSecs: Int = 5
```

Comment:
I had used every 20 seconds in the smoke tests, because the query itself seemed costly.  But that might not be true here- this is a pretty simple query.

Reply:
Super simple small scoped query since it is created_at bound.